### PR TITLE
(maint) Add Docker build version support

### PR DESCRIPTION
**Pull request by Iristyle**
- Use the git describe as we're doing in Bash scripts to pull a version
   from the repository when building / testing Docker containers

### CODE REVIEW

File: docker/ci/build.ps1 [hunk 1]
```diff
@@ -19,14 +32,14 @@ function Build-Container($Name, $Repository = '127.0.0.1')
 {
   Push-Location (Join-Path (Get-CurrentDirectory) '..')
   bundle exec puppet-docker local-lint $Name
-  bundle exec puppet-docker build $Name --no-cache --repository $Repository --build-arg namespace=$Repository
+  bundle exec puppet-docker build $Name --no-cache --repository $Repository --version $Version --no-latest --build-arg namespace=$Repository
```
Comment (underscorgan) [inline docker/ci/build.ps1:35]: Missing  $Version parameter in function def

File: docker/ci/build.ps1 [hunk 2]
```diff
@@ -19,14 +32,14 @@ function Build-Container($Name, $Repository = '127.0.0.1')
...
   Pop-Location
 }
 
 function Invoke-ContainerTest($Name, $Repository = '127.0.0.1')
 {
   Push-Location (Join-Path (Get-CurrentDirectory) '..')
-  bundle exec puppet-docker spec $Name --image $Repository/$Name
+  bundle exec puppet-docker spec $Name --image "$Repository/${Name}:$Version"
```
Comment (underscorgan) [inline docker/ci/build.ps1:42]: Missing $Version parameter in function def

**Review by underscorgan — CHANGES_REQUESTED** [review]

**Review by underscorgan — APPROVED** [review]

File: docker/ci/build.ps1 (hunk 1 above)
Comment (Iristyle) [reply docker/ci/build.ps1:35]: jinx ;0

Comment (Iristyle) [reply docker/ci/build.ps1:35]: I don't know what's wrong with me today!

**Pull request merged**